packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -0,0 +1,119 @@
+
+    return (
+      <section className={classes} aria-label={t.bannerLabel} lang={langProp}>
+        <header
+          className={`ds-c-usa-banner__header ${
+            this.state.isBannerOpen ? 'ds-c-usa-banner__header--expanded' : ''
+          }`}
+        >
+          <p className="ds-c-usa-banner__header-text">
+            <IconFlagComponent
+              role="img"
+              className="ds-c-usa-banner__header-flag"
+              focusable="false"
+            />
+          </p>
+          <p className="ds-c-usa-banner__header-text">
+            <span>{t.bannerText}</span>
+            <span className="ds-c-usa-banner__header-action" aria-hidden="true">
+              {t.bannerActionText}
+            </span>
+            <button
+              onClick={this.handleToggleBanner}
+              className="ds-c-usa-banner__button"
+              aria-expanded={this.state.isBannerOpen}
+              aria-controls={this.id}
+            >
+              <span className="ds-c-usa-banner__button-text">{t.bannerActionText}</span>
+            </button>
+          </p>
+        </header>
+        <div className="ds-c-usa-banner__content" id={this.id} hidden={!this.state.isBannerOpen}>
+          <div className="ds-u-display--flex ds-u-flex-direction--column ds-u-sm-flex-direction--row ds-u-flex-wrap--nowrap">
+            <div className="ds-c-usa-banner__guidance">
+              <IconDotGov className="ds-c-usa-banner__icon" aria-hidden="true" />
+              <p className="ds-c-usa-banner__media-body">
+                <strong>{t.domainHeaderText}</strong>
+                <br />
+                {t.domainAText}
+                <strong> {t.govText} </strong>
+                {t.domainText}
+              </p>
+            </div>
+            <div className="ds-c-usa-banner__guidance">
+              <IconHttps className="ds-c-usa-banner__icon" aria-hidden="true" />
+              <p className="ds-c-usa-banner__media-body">
+                <strong>{t.httpsHeaderText}</strong>
+                <br />
+                {t.httpsAText}
+                <strong> {t.httpsLockText} </strong> ({' '}
+                <IconLockComponent
+                  role="img"
+                  className="ds-c-usa-banner__lock-image"
+                  focusable="false"
+                />{' '}
+                ) {t.httpsOrText}
+                <strong> {t.httpsText} </strong>
+                {t.httpsDetailText}
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+    );
+  }
+}
+
+UsaBanner.defaultProps = {
+  locale: 'en',
+};
+
+UsaBanner.propTypes = {
+  /**
+   * Additional classes to be added to the root `section` element
+   */
+  className: PropTypes.string,
+  /**
+   * A unique ID to be applied to the banner content. A unique ID will be generated if one isn't provided.
+   */
+  id: PropTypes.string,
+  /**
+   *
+   * The language the USA Banner will render as.
```

Comment:
Update the text here to read a bit more friendly 

```suggestion
   * The language the USA Banner will be presented in.
```